### ISSUE TITLE
Verify Messages for Audio

### ISSUE THREAD

**Issue by jag3773** [unfoldingWord-dev/uw-android#172]
There should be no verification icons/information for audio files.  Right now both iOS and Android seem to indicate that the audio is not verified (failed verification), which is not the case.  Would be much better to show no verification information for audio.

**Comment by bknatterud** [comment]
But what is the point of verification if we are not going to show it is failing?

**Comment by dwsolberg** [comment]
@jag3773 See Bryn's note. Right now, there is no signature for the files, which means they are not verified. There is no way for the app to distinguish between files that should have no signature and those that should (otherwise, one could easily fake verification by deleting the files).

**Comment by jag3773** [comment]
It is not failing, we are simply not signing mp3s.  Disabling verification on mp3 files seems like a pretty small task.

**Comment by bknatterud** [comment]
I don't think it is so much the size of the task as it is just not a good idea to have exceptions for verification. But maybe what you are suggesting is to disable all verification for audio files? Is what you are thinking?

**Comment by jag3773** [comment]
Yes, disable all verification for audio files.

**Comment by PeeJWeeJ** [comment]
Removed on android
